src/components/SidebarState.vue

```diff
@@ -15,13 +15,13 @@ export default Vue.extend({
     },
     computed: {
         isDrawn() {
-            return this.sidebarOpen && this.$state.ui.app_width <= 769;
+            return this.sidebarOpen && this.section() && this.$state.ui.app_width <= 769;
         },
         isClosed() {
-            return !this.sidebarOpen;
+            return !this.section();
```

Comment:
Everywhere else checks for `this.sidebarOpen`, shouldn't this do the same?

Reply:
seems this computed was no longer used